src/connectionconfig/connectionDialogWebviewController.ts

```diff
@@ -734,6 +766,26 @@ export class ConnectionDialogWebviewController extends FormWebviewController<
         return state;
     }
 
+    private removeDefaultProperties(
+        connProfile: IConnectionProfile,
+    ): IConnectionProfile {
+        // TODO: ideally this compares against the default values acquired from a source of truth (e.g. STS),
+        // so that it can clean up more than just undefined properties.
```

Comment:
perhaps the name of the method should better reflect the default, i.e., removeUndefinedProperties or something like that?